packages/tangle/booker.go

```diff
@@ -51,7 +51,7 @@ func NewBooker(tangle *Tangle) (messageBooker *Booker) {
 
 // Setup sets up the behavior of the component by making it attach to the relevant events of other components.
 func (b *Booker) Setup() {
-	b.tangle.Orderer.Events.MessageOrdered.Attach(events.NewClosure(func(messageID MessageID) {
+	b.tangle.Solidifier.Events.MessageSolid.Attach(events.NewClosure(func(messageID MessageID) {
```

Comment:
As we typically have multiple go-routines that deal with solidification, wouldn't these go-routines then also continue the execution of the booking?
I think the flow should be interrupted here and have the booker running with only 1 single go-routine. WDYT?